Update help files for {Update, Find}-PSResource and {Register, Set, Get, Unregister}-PSResourceRepository (#434)
PowerShell/PowerShellGet: help/Find-PSResource.md

### -Repository
Specifies one or more repository names to search.

> If not specified, search will include all currently registered repositories, in order of highest priority, til we reach the first repository the package is found in.

Reviewer: ```suggestion
If not specified, search will include all currently registered repositories, in order of highest priority, until a repository is found that contains the package.
```
